pkg/apis/kops/model/utils.go

```diff
@@ -43,6 +44,13 @@ func FindZonesForInstanceGroup(c *kops.Cluster, ig *kops.InstanceGroup) ([]strin
 
 		if subnet.Zone != "" {
 			zones.Insert(subnet.Zone)
+			continue
+		}
+
+		// fallback to setting the Region
+		if subnet.Region != "" {
```

Comment:
The reason this is going wrong is that for GCE the Subnets are regional, but then each IG has Zones.

I guess we need `FindRegionsForInstanceGroup` and then DO can switch each usage to that?   That's going to be a little tedious - but maybe not _too_ bad given there are only 5 or so places?

Question though: are NYC1, NYC2 and NYC3 not the equivalent of AZs?  Or is it just that because volumes aren't zonal, there's not much point in the concept of zones on DO?